Open Judge System/Web/OJS.Web/Content/Site.css

```diff
@@ -351,3 +351,7 @@ pre {
 .pagination > li > span:focus {
     background-color: #0f7864;
 }
+
+.inline-form {
```

Comment:
Bootstrap does indeed have a similar class, namely "form-inline". However it doesn't display the form inline appropriately.